### ISSUE TITLE
Nokogiri warnings with latest vagrant

### ISSUE THREAD

**Issue by ggascoigne** [mitchellh/vagrant#8581]
Nokogiri warnings generated on every command

`WARNING: Nokogiri was built against LibXML version 2.9.0, but has dynamically loaded 2.9.2`

### Vagrant version
Vagrant 1.9.4

### Host operating system
OSX 10.11.6

### Guest operating system
doesn't matter

### Vagrantfile
again, doesn't seem to matter

### Expected behavior

no warning should be issued

### Steps to reproduce
1. create a fresh vagrant project `vagrant init some_box_name`
2. ```bash
$ vagrant status
WARNING: Nokogiri was built against LibXML version 2.9.0, but has dynamically loaded 2.9.2
==> default: Box 'some_box_name' could not be found. Attempting to find and install...
    default: Box Provider: vsphere
    default: Box Version: >= 0
==> default: Box file was not detected as metadata. Adding it directly...
==> default: Adding box 'contos' (v0) for provider: vsphere
    default: Downloading: some_box_name
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.
```

I'd like to find a fix for this that doesn't mean hacking into the vagrant distribution.

**Comment by cblecker** [comment]
I think this was fixed in https://github.com/mitchellh/vagrant/pull/8571

**Comment by chrisroberts** [comment]
Fixed via #8571
